_posts/2024-01-29-welcome-to-jekyll.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "Welcome to Jekyll!"
```

Comment:
Can we remove this file perhaps?

Reply:
Oh, yeah.  That slipped under the radar. It's part of the boilerplate.  I will remove it.

Reply:
It's been removed.